src/dgipy/dgidb.py

```diff
@@ -263,15 +265,15 @@ def get_clinical_trials(
                         "minimumAge"
                     ]
                 except:
-                    new_row["min_age"] = "N/A"
+                    new_row["min_age"] = None
 
                 new_row["age_groups"] = study["protocolSection"]["eligibilityModule"][
                     "stdAges"
                 ]
                 if "CHILD" in new_row["age_groups"]:
-                    new_row["Pediatric?"] = "Yes"
+                    new_row["Pediatric?"] = True
                 else:
-                    new_row["Pediatric?"] = "No"
+                    new_row["Pediatric?"] = False
```

Comment:
double check me on the syntax but this can be `new_row["Pediatric?"] = "CHILD" in new_row["age_groups"]`

Reply:
This checks out